src/content/en/updates/2021/02/css-in-js.md

```diff
@@ -0,0 +1,149 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: How we support CSS-in-JS in DevTools and how it is different from regular CSS.
+
+{# wf_updated_on: 2021-02-26 #}
+{# wf_published_on: 2021-02-26 #}
+{# wf_tags: devtools-blog #}
+{# wf_featured_image: /web/updates/images/2021/02/css-in-js/devtools-blog.jpg #}
+{# wf_featured_snippet: How we support CSS-in-JS in DevTools and how it is different from regular CSS. #}
+{# wf_blink_components: N/A #}
+
+# CSS-in-JS support in DevTools {: .page-title }
+
+{% include "web/_shared/contributors/orkon.html" %}
+
+<<../../_shared/devtools-research.md>>
+
+This article talks about CSS-in-JS support in DevTools that landed a year ago and, in general, what we mean by CSS-in-JS and how it’s different from regular CSS that has been supported by DevTools for a long time.
```

Comment:
Is it really already a year?

Would it not be better if we use the Chrome release, e.g. Chrome 85 or something? That way the release date of the blog post doesn't matter.

Reply:
Updated